auto_nag/bzcleaner.py

```diff
@@ -521,7 +521,7 @@ def set_needinfo(self):
                         extra=self.get_extra_for_needinfo_template(),
                         plural=utils.plural,
                         bugid=bugid,
-                        documentation=doc,
+                        documentation=doc
```

Comment:
This will fail the linter:
```suggestion
                        documentation=doc,
```
You could use pre-commit to catch such things earlier:
```sh
$ pre-commit install
```